### PR TITLE
add: health-check for express middleware.

### PR DESCRIPTION
**Pull request by PiusLucky**

### CODE REVIEW

File: s/api/express-middleware.ts [hunk 1]
```diff
@@ -43,6 +43,12 @@ export async function expressMiddleware({
 	const execute = renraku.servelet(api)
 
 	const middleware: express.RequestHandler = async(req, res, next) => {
+		if (req.url.endsWith("/health")) {
+			console.log(`⚕️ health check`)
+			res.setHeader("Content-Type", "text/plain; charset=utf-8")
+			res.statusCode = 200
+			res.end(Date.now().toString())
+		}
 		const userIntegration = await authUser(req)
```
Comment (chase-moskal) [inline s/api/express-middleware.ts:52]: oh, you need to wrap the rest of the code in the middleware in an `else` block

**Review by chase-moskal — CHANGES_REQUESTED** [review]

File: s/api/express-middleware.ts (hunk 1 above)
Comment (PiusLucky) [reply s/api/express-middleware.ts:52]: okay. I will implement that now.

File: s/api/express-middleware.ts [hunk 2]
```diff
@@ -43,19 +43,27 @@ export async function expressMiddleware({
...
-		const meta: Meta = {
-			user: userIntegration
-				? {...userIntegration, id: userIntegration.id.string}
-				: undefined,
+		if (req.url.endsWith("/health")) {
+			console.log(`⚕️ health check`)
+			res.setHeader("Content-Type", "text/plain; charset=utf-8")
+			res.statusCode = 200
+			res.end(Date.now().toString())
+		}else {
```
Comment (chase-moskal) [inline s/api/express-middleware.ts:51]: whitespace is sub-par on this line — but i'll let it slide, this time ;)

File: s/api/express-middleware.ts (hunk 1 above)
Comment (PiusLucky) [reply s/api/express-middleware.ts]: Done

**Comment by chase-moskal** [comment]
added development link to #91

**Pull request merged**